.github/workflows/test.yml

```diff
@@ -9,23 +9,9 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ['3.9']
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
-        run: |
-          pip install -U pip setuptools wheel
-          pip install -r ./requirements.txt
-
-      - name: Run pre-commit
-        run: pre-commit install && pre-commit run --all-files
+    uses: less-action/reusables/.github/workflows/pre-commit.yaml@v1
```

Comment:
Can you please pin this to some existing commit hash?

Reply:
Can do! 👍